src/EditorFeatures/CSharpTest/GenerateDefaultConstructors/GenerateConstructorForAbstractClass.cs

```diff
@@ -0,0 +1,318 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.GenerateDefaultConstructors;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.GenerateDefaultConstructors
+{
+    public class GenerateConstructorForAbstractClassTests : AbstractCSharpCodeActionTest
```

Comment:
Thanks for the PR :)

we don't need a new test class for these tests.  It should just go in the primary test class htat already exists for this feature.  You can put it in another partial-part if you want to (though i would not recommend that personally).

Reply:
Thank you for Code Review :+1: 
Done.